edgelet/iotedge-proxy/src/routine.rs

```diff
@@ -9,7 +9,7 @@ use futures::sync::oneshot::Receiver;
 use futures::{Future, IntoFuture};
 use hyper::Server;
 use log::{debug, info, warn};
-use tokio::runtime::current_thread::Runtime;
```

Comment:
Your previous explanation seemed reasonable, what changed?

Reply:
This isn't a test.